docs/source/asr/speaker_diarization/configs.rst

```diff
@@ -18,12 +15,12 @@ Example configuration files for speaker diarization can be found in the
 
 
 Dataset Configuration
------------------------
+---------------------
 
 In contrast to other ASR related tasks or models in NeMo, speaker diarization supported in NeMo is a modular inference pipeline.
 Datasets here denotes the data you would like to perform speaker diarization on. 
 
-An example Speaker Diarization dataset configuration could look like:
+An example Speaker Diarization dataset Hydra configuration could look like:
 
 .. code-block:: yaml
 
```

Comment:
This is an outdated yaml config. remove or update accordingly

Reply:
updated this part. this was a crucial mistake.